common/src/autogluon/common/savers/save_pd.py

```diff
@@ -13,7 +14,52 @@
 
 # TODO: Update so verbose prints at level 20, and adjust calls to save accordingly
 # gzip compression produces random deflate issues on linux machines - use with caution
-def save(path, df, index=False, verbose=True, type=None, sep=',', compression='gzip', header=True, json_dump_columns=None):
+# TODO: v1.0 deprecate 'df', replace with 'data', or align with Pandas for parameter names
+# TODO: v1.0 consider renaming function so it isn't 'save'. Consider instead 'save_pd', or something more descriptive.
+# TODO: Add full docstring
+# TODO: Add `allow_overwrite=True` so that users can instead force
```

Comment:
`False` by default.

Reply:
That would be a 1.0 change since it is highly breaking. Also, I don't recommend allow_overwrite=False by default, it would cause many problems.